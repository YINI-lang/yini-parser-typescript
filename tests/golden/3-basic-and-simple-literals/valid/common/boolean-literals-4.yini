^ Booleans
bool1 = tRUE
bool2 = fALSE
bool3 = oN
bool4 = oFF
bool5 = yES
bool6 = nO
