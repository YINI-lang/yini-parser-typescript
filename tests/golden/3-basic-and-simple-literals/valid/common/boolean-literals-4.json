{
    "Booleans": {
        "bool1": true,
        "bool2": false,
        "bool3": true,
        "bool4": false,
        "bool5": true,
        "bool6": false
    }
}
